pyppeteer/frame/__init__.py

```diff
@@ -31,7 +32,7 @@ def __init__(
         self._lifecycleEvents: Set[str] = set()
         self._mainWorld = DOMWorld(frameManager, self, frameManager._timeoutSettings)
         self._secondaryWorld = DOMWorld(frameManager, self, frameManager._timeoutSettings)
-        self._childFrames: Set[Frame] = set()  # maybe list
+        self._childFrames: OrderedSet[Frame] = OrderedSet()
```

Comment:
There was an issue on the orderedset project asking about type hinting, and it implied this currently isn't possible. I think it's fine to keep it in for now though, there's hundreds of mypy errots and we can easily change this one in a batch of mypy fixes.